envoy/cilium_network_policy.h

```diff
@@ -152,17 +163,28 @@ class NetworkPolicyMap : public Singleton::Instance,
       }
 
       bool Matches(uint32_t port, uint64_t remote_id, const Envoy::Http::HeaderMap& headers) const {
+	bool found_port_rule = false;
```

Comment:
I don't understand why this is necessary.
I think the case when the port was not found was already handled by the final `return` at the bottom.
That `return` is hit only when the port is not found.
The change was already correct, you only need to change the value of the last `return` to `true`.